app/controllers/application_controller.rb

```diff
@@ -39,7 +39,7 @@ def reject_if_banned
 
   def current_user
     if session[:user_id]
-      @current_user ||= User.find(session[:user_id])
+      @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
Did you change this so it 404's? If so, is that better than 500?

Reply:
I changed this because it is apparently possible for a Remember token to remain valid even after the deletion of a user, which was causing some tests to malfunction by attempting to log in as a non-existent user. After typing the previous sentence out, I think we should probably do something about that.

Reply:
Hmm remember tokens are destroyed when the user is destroyed, I wonder what's going on there

Reply:
If that's the case, maybe there's something fishy happening with the session token itself being retained? We can chat more on Discord for faster communication.

Reply:
Resolved on Discord. Issue being tracked in #140